testing/skia_gold_client/lib/skia_gold_client.dart

```diff
@@ -495,5 +481,12 @@ class SkiaGoldClient {
   }
 }
 
+Future<void>? _oneResult;
```

Comment:
I don't think we can make this a global, it needs to be per `SkiaGoldClient`. We have separate gold clients per suite now.

Reply:
Done.

Nice catch! I've been contemplating making it per client instead of global but didn't find a good reason to do that. Totally forgot that we run multiple suites at the same time now.